draft-ietf-quic-qpack.md

```diff
@@ -1014,12 +1014,12 @@ represented as an 8-bit prefix string literal.
 QPACK defines two settings which are included in the HTTP/QUIC SETTINGS frame.
 
   SETTINGS_HEADER_TABLE_SIZE (0x1):
-  : An integer with a maximum value of 2^30 - 1.  The default value is 4,096
+  : An integer with a maximum value of 2^30 - 1.  The default value is zero
     bytes.  See {{table-dynamic}} for usage.
```

Comment:
This is an interesting question.  [Section 2.2.2 of QPACK-03](https://tools.ietf.org/html/draft-ietf-quic-qpack-03#section-2.2.2) begins as follows:

>  The size of the dynamic table is the sum of the size of its entries.
>
>  The size of an entry is the sum of its name's length in octets (as defined in Section 5.1.2), its value's length in octets, and 32.

My reading of this is that the table size is measured in octets.

Reply:
Units to be used in interpreting a setting need to be in the definition of the setting.  The alternative, I suppose, would be to move the units to the section that defines how the integer is used, but I imagine someone looking to see what to put here would appreciate knowing whether to send 65.535 or 64.

Reply:
Also, it appears that @martinthomson's change to replace octets with bytes last month was global, so that text doesn't match the current editor's copy of QPACK.